apps/web/lib/teams/teams.ts

```diff
@@ -15,3 +16,19 @@ export const useTeam = (environmentId: string) => {
     mutateTeam: mutate,
   };
 };
+
+export const addDemoProduct = async (teamId: string) => {
```

Comment:
@pandeymangg Is this function still being used somewhere?

Reply:
not right now, forgot to remove it from the code, will do it ☺️